pkg/core/exec.go

```diff
@@ -27,13 +28,15 @@ func Exec(ctx context.Context, command string, source string, target string) (ou
 	defer cancel()
 
 	collEnv := collations.NewEnvironment(mysqlVersion)
-	parser, err := sqlparser.New(sqlparser.Options{
+	vtenv, err := vtenv.New(vtenv.Options{
 		MySQLServerVersion: mysqlVersion,
+		TruncateUILen:      servenv.TruncateUILen,
```

Comment:
We should not use servenv here but provide these values ourselves. 

The defaults of 0 are actually fine and better. It also will clean up go.mod again and greatly reduce the binary size.